g2core/stepper.h

```diff
@@ -337,7 +337,7 @@ typedef enum {
  *  There are 5 main structures involved in stepper operations;
  *
  *  data structure:                   found in:      runs primarily at:
- *    mpBuffer planning buffers (bf)    planner.c      main loop
+ *    mpBuffer planning buffers (bf)    planner.c       main loop
```

Comment:
why change alignment?

Reply:
I suspect this wasn't intentional, but I don't recall.